evento/app/controllers/users_controller.rb

```diff
@@ -15,13 +16,17 @@ def show
 
   # GET /users/1/events
   def events
-    render json: @user.events
+    if @current_user == @user
+      render json: @user.events
+    else
+      render json: { error: 'Not Authorized' }, status: 401
+    end
   end
 
   # POST /users
   def create
     @user = User.new(user_params)
-
+    
     if @user.save
       render json: @user, status: :created, location: @user
     else
```

Comment:
`create` does not work.  It returns `{ "password": ["can't be blank" ] }`. First I thought it was because `user_params` did not allow `:password` but it did not fix it